dashboard/app/models/parent_levels_child_level.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: parent_levels_child_levels
+#
+#  id              :integer          not null, primary key
+#  parent_level_id :integer          not null
+#  child_level_id  :integer          not null
+#  position        :integer
```

Comment:
Do we need to order on position in the associations in Level? (I know I argued for doing it in the code instead of in the association last time, but if we're already doing it this way throughout our code base, it's probably better to be consistent. I've come around.)

Reply:
Sounds great, I am glad you are on board with this pattern! I'll add it.